README.md

```diff
@@ -110,6 +110,12 @@ Valid values are:
 
 Older versions of WordPress will not run on PHP7, see this page on [how to change PHP version per site](https://varyingvagrantvagrants.org/docs/en-US/adding-a-new-site/changing-php-version/).
 
+```yaml
+custom:
+    force_wp_version: true
```

Comment:
maybe is not so clear that is for upgrade? Maybe something like `force_chage_wp_version`?